app.json

```diff
@@ -67,6 +67,12 @@
           "microphonePermission": false
         }
       ],
+      [
+        "expo-screen-orientation",
+        {
+          "initialOrientation": "DEFAULT"
+        }
```

Comment:
does this make our orientation provider superfluous?

Reply:
good info.
1. buttons should not be full width on large screens. that is too large.
2. according full width of contents we decided, that we do not want to smallen anymore, right?!
so we still need the provider but maybe need some adjustments?
but this should go in a separate ticket and process.

Reply:
i don't have in mind if it's just about the button or maybe anything else. the carousel image is also something i can think of.

Reply:
can you please check if there is anything else orientation aware like the button? the carousel or html image on top of screens?

Reply:
and what about androids input change on landscape? is this normal?

|portrait|landscape|
|---|---|
|![Screenshot_1681733850](https://user-images.githubusercontent.com/1942953/232482026-db4c3be0-5e46-4079-aad4-0a89550078df.png)|![Screenshot_1681733863](https://user-images.githubusercontent.com/1942953/232482053-21037562-ed5f-4524-8f4f-dbf038da2afd.png)|

do we and the users want/expect it the same on landscape as on portrait?

Reply:
In this version we use `WrapperWithOrientation` component. In version `3.2.1` this component will be removed. We use `WrapperWithOrientation` component in many places but `noFlex` property is used only by `TextBlock`. We use `needLandscapeStyle` for Landscape. This is used by `Button`, `Checkbox` and `OrientationAwareIcon`.

---
I think it is caused by the `react-native-elements` update. If you try it in version `3.1.3`, it looks the same.
I have encountered Input in this way in other Android applications.

Reply:
Is there a prop to avoid that?

Reply:
I think this little code will solve our problem. 👍
<img width="175" alt="image" src="https://user-images.githubusercontent.com/11755668/237045310-62101a85-af1a-446b-9e9b-55f4df5817ed.png">

Reply:
and how does it look like now with the change? what is the effect?

Reply:
Looks the same as before.